### PR TITLE
[Dani Alcala] slider tests

### PR DESCRIPTION
**Pull request by daniamcode**

### CODE REVIEW

File: fenizia-react/src/actions/productActions.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ import productList from "../product.mock.js";
 import dispatcher from "../appDispatcher";
 import actionTypes from "./actionTypes";
 
+// export async function loadProducts() {
+//   const product = await dispatcher.dispatch({
+//     type: actionTypes.LOAD_PRODUCTS,
+//     data: product,
+//   });
+// }
+
```
Comment (simonbesteiro) [inline fenizia-react/src/actions/productActions.js]: remove commented code

File: fenizia-react/src/actions/productActions.js [hunk 2]
```diff
@@ -13,12 +20,12 @@ export function loadProducts() {
...
-}
+// export function NextCarousel() {
+//   return new Promise((resolve) => {
+//     resolve();
+//   }).then(() => {
+//     dispatcher.dispatch({
+//       type: actionTypes.NEXT_CAROUSEL,
+//     });
+//   });
+// }
```
Comment (simonbesteiro) [inline fenizia-react/src/actions/productActions.js]: remove commented code

File: fenizia-react/src/stores/productStore.js [hunk 3]
```diff
@@ -34,15 +33,15 @@ class ProductStore extends EventEmitter {
...
-    if (_product && _product.length >= calculus) {
-      return _product.slice(calculus - step, calculus);
-    } else return _product.slice(_product.length - ++page, _product.length);
-  }
+  // getCarouselPage(page, step) {
+  //   const calculus = step * page;
+  //   if (_product && _product.length >= calculus) {
+  //     return _product.slice(calculus - step, calculus);
+  //   } else return _product.slice(_product.length - ++page, _product.length);
+  // }
```
Comment (simonbesteiro) [inline fenizia-react/src/stores/productStore.js]: remove commented code

File: fenizia-react/src/stores/productStore.js [hunk 4]
```diff
@@ -52,9 +51,9 @@ dispatcher.register((action) => {
       _product = action.data;
       productStore.emitChange(_product);
       break;
-    case actionTypes.NEXT_CAROUSEL:
-      productStore.emitChange();
-      break;
+    // case actionTypes.NEXT_CAROUSEL:
+    //   productStore.emitChange();
+    //   break;
```
Comment (simonbesteiro) [inline fenizia-react/src/stores/productStore.js]: remove commented code

**Review by simonbesteiro — CHANGES_REQUESTED** [review]
remove commented code

File: fenizia-react/src/actions/productActions.js (hunk 1 above)
Comment (daniamcode) [reply fenizia-react/src/actions/productActions.js]: done

File: fenizia-react/src/actions/productActions.js (hunk 2 above)
Comment (daniamcode) [reply fenizia-react/src/actions/productActions.js]: done

File: fenizia-react/src/stores/productStore.js (hunk 3 above)
Comment (daniamcode) [reply fenizia-react/src/stores/productStore.js]: done

File: fenizia-react/src/stores/productStore.js (hunk 4 above)
Comment (daniamcode) [reply fenizia-react/src/stores/productStore.js]: done

**Pull request merged**